module/zfs/zfs_ioctl.c

```diff
@@ -5037,6 +5038,18 @@ zfs_ioc_pool_reopen(zfs_cmd_t *zc)
 {
 	spa_t *spa;
 	int error;
+	nvlist_t *reopenflags;
+	boolean_t no_scrub_restart;
+
+	if (zc->zc_nvlist_src != 0 &&
+	    (error = get_nvlist(zc->zc_nvlist_src, zc->zc_nvlist_src_size,
+	    zc->zc_iflags, &reopenflags)) != 0)
+		return (error);
+
+	error = nvlist_lookup_boolean_value(reopenflags,
+	    "no_scrub_restart", &no_scrub_restart);
+	if (error != 0)
+		return (SET_ERROR(EINVAL));
```

Comment:
Making this fatal isn't great since it means no older versions of `zpool(8)` will be able to run the `zpool reopen` command with newer modules.  When the nvlist isn't present we should  default to the safe behavior as if the option wasn't provided.

Reply:
I thought that the user space tools should go in pair with the kernel modules. I'll fix this. 
When I do bisect the zpool import almost never works when there is a large range between commits. If the user can't import pool with the older tools it won't be able to reopen it neither.

Reply:
They're best kept in sync, but we do try our best to avoid any unnecessary incompatibilities.